graphql/resolvers.go

```diff
@@ -1060,6 +1060,18 @@ func (r *queryResolver) TaskTests(ctx context.Context, taskID string, execution
 	if dbTask == nil || err != nil {
 		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("cannot find task with id %s", taskID))
 	}
+	baseTask, err := dbTask.FindTaskOnBaseCommit()
+	if err != nil {
+		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("cannot find base task with id %s", taskID))
```

Comment:
I think InternalServerError is more suitable here. It is possible that the resource isn't found and an error is not thrown on line 1069